lebedigital/calibration/calibrationWorkflow.py

```diff
@@ -0,0 +1,202 @@
+# third party imports
+import os
+from pathlib import Path
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import matplotlib as mpl
+from matplotlib import rc
+
+mpl.rcParams['font.size'] = 16
+rc('text', usetex=True)
+
+# local imports (probeye)
+from probeye.definition.inverse_problem import InverseProblem
+from probeye.definition.likelihood_model import GaussianLikelihoodModel
+from probeye.ontology.knowledge_graph_export import export_knowledge_graph
+from probeye.ontology.knowledge_graph_import import import_parameter_samples
+from probeye.ontology.knowledge_graph_export import export_results_to_knowledge_graph
+from probeye.inference.emcee.solver import EmceeSolver
+
+# local imports (others)
+from lebedigital.Prediction.three_point_bending_example import three_point_bending_example
+from lebedigital.calibration.utils import query_KG, PosteriorPredictive, load_experimental_data
+from lebedigital.calibration.forwardmodel_linear_elastic_cylinder import LinearElasticityCylinder
+
+
+def perform_calibration(path_to_KG : str, calibrated_data_path : str, experiment_name='Wolf 8.2 Probe 1', mode = 'full', KnowledgeGraph = True):
+    # =========================================
```

Comment:
The output of the method should also be documented.

Reply:
Yes alright.